src/integrators/verner_rk_integrators.jl

```diff
@@ -153,6 +161,12 @@ end
   integrator.kshortsize = 10
   k = eltype(integrator.sol.k)(integrator.kshortsize)
   integrator.k = k
+
+  # Avoid undefined entries if k is an array of arrays
+  prototype = f(integrator.t, integrator.uprev)
```

Comment:
If `eltype(integrator.sol.k)` is e.g. `Array{Float64,1}` we do only know the type of the elements of `k` but not their length. So I thought the safest way would be to use the result of a function evaluation as a prototype since `k` is filled with results of such function evaluations later.